src/startup.js

```diff
@@ -7,6 +7,8 @@ import seedEmailTemplatesForShop from "./util/seedEmailTemplatesForShop.js";
  * @returns {undefined}
  */
 export default async function emailTemplatesStartup(context) {
+  await seedEmailTemplatesForShop(context, "CegM7hSbdShjomCsD");
```

Comment:
This should be a shop._id variable, not a hard-coded ID.

Reply:
Wow, sorry about this! Debugging code slipped through in my commits. I'll remove it.